pkg/client/chains.go

```diff
@@ -0,0 +1,133 @@
+package client
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	testnetURL = "https://testnet.tableland.network"
+	localURL   = "http://localhost:8080"
```

Comment:
Once we launch a mainnet validator, will add it's URL here and update the mainnet EVM `Chain`s below to use that URL. Although, once there are many validators, we'll have to revisit this specifying of URL all together (as mentioned in my previous PR).